src/aura/allContacts/allContactsHelper.js

```diff
@@ -1,4 +1,16 @@
 ({
-	helperMethod : function() {
+	loadContacts : function(component, accountId) {
+		var action = component.get("c.findContactsByAccount");
```

Comment:
this name could be just getContactsByAccount, a super small detail but devil is on details

Reply:
Done